src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -29,6 +38,16 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<ICachedSecretProvider>(serviceProvider => new CachedSecretProvider(secretProvider: null));
 #endif
 
+#if CertificateAuth
+            #error Please provide a valid certificate issuer name for the client certificate authentication
+            var certificateAuthenticationConfig = 
+                new CertificateAuthenticationConfigBuilder()
+                    .WithSubject(X509ValidationLocation.SecretProvider, "YOUR KEY TO CERTIFICATE SUBJECT NAME")
```

Comment:
Just wondering, should we stimulate moving certificate subject name into key vault? Doesn't feel like a secret to me, no?

Reply:
It depends on what the user-defined `ISecretProvider` does; by default; we don't use Key Vault of course.

Reply:
Yes but it would still be good to switch to configuration as you're basing that off of an assumption and we should be a good citizen and show that config is better for this kind of info imo

Reply:
Ah, ok I think I get it; but could you make an issue for this? Thanks.